<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BooksManage</class>
 <widget class="QWidget" name="BooksManage">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_info">
     <layout class="QHBoxLayout" name="horizontalLayout_info">
      <item>
       <layout class="QGridLayout" name="gridLayout_form">
        <item row="0" column="0">
         <widget class="QLabel" name="label_cardID">
          <property name="text">
           <string>卡号：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_cardID"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_goodsID">
          <property name="text">
           <string>书籍编号：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEdit_goodsID"/>
        </item>

        <item row="1" column="0">
         <widget class="QLabel" name="label_name">
          <property name="text">
           <string>书名：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_name"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_author">
          <property name="text">
           <string>作者：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="lineEdit_author"/>
        </item>

        <item row="2" column="0">
         <widget class="QLabel" name="label_publisher">
          <property name="text">
           <string>出版社：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_publisher"/>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_type">
          <property name="text">
           <string>图书类型：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="lineEdit_type"/>
        </item>

        <item row="3" column="0">
         <widget class="QLabel" name="label_days">
          <property name="text">
           <string>可借天数：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_days"/>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_publish_date">
          <property name="text">
           <string>出版时间：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QDateEdit" name="dateEdit_publish">
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <item>
      <widget class="QPushButton" name="pushButton_add">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_update">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_search">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_table">
     <property name="title">
      <string>图书列表</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_table">
      <item>
       <widget class="QTableWidget" name="tableWidget_books"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
